client/lib/store-transactions/index.js

```diff
@@ -349,6 +349,7 @@ TransactionFlow.prototype._submitWithPayment = function ( payment ) {
 	const transaction = {
 		cart: omit( this._initialData.cart, [ 'messages' ] ), // messages contain reference to DOMNode
 		domain_details: this._initialData.domainDetails,
+		account_email: this._initialData.accountEmail,
```

Comment:
You probably are aware, but just for the record this change applies only to old checkout.

Reply:
You're probably right.  But the thing is, this is the only place in the pull request where `account_email` appears... so if this isn't being used to send an `account_email` parameter to the API endpoint, then it's not clear to me what is (the API endpoint definitely does receive that).  But there may be something I missed... and to be honest, I never tried removing this line either :smile:

Reply:
If the data is part of the data objects that you added it to, it will be sent to the endpoint automatically. 👍